test/com/facebook/buck/android/testdata/android_project/scala/com/sample/lib/BUCK.fixture

```diff
@@ -0,0 +1,35 @@
+android_library(
+  name = 'lib',
+  srcs = glob(['*.scala']),
+  language = 'SCALA',
```

Comment:
Added in `AndroidLibraryIntegrationTest`, although I'm not sure if that's exactly right, this fails on Arg parsing level, so it's not reported as build failure, but `runBuckBuild` throws an exception.